test/TicketingService.Storage.PgSqlMarten.Tests/MartenTicketingTests.cs

```diff
@@ -44,7 +44,7 @@ public async Task CreateGetUpdatePendingErrorCompleteDelete()
 
             // error
             var ticketError = new TicketError("mockErrorMessage", "mockErrorCode");
-            await ticketing.Error(ticketId, ticketError);
+            await ticketing.Error(ticketId, new[] { ticketError });
```

Comment:
Can you expand the test by adding multiple ticketErrors and than assert on retrieving them again?